### ISSUE TITLE
Cannot extend exported TS namespace member

### ISSUE THREAD

**Issue by benmerckx** [oven-sh/bun#2826]
### What version of Bun is running?

0.5.9

### What platform is your computer?

Microsoft Windows NT 10.0.19044.0 x64

### What steps can reproduce the bug?

````ts
namespace Test {
  export class Exported {
    constructor() {}
  }
  class Sub extends Exported {}
}
````

Will result in

````
$ bun test.ts

1 | namespace Test {
2 |   export class Exported {
3 |     constructor() {}
4 |   }
5 |   class Sub extends Exported {}
      ^
TypeError: The superclass is not a constructor.
      at /test.ts:5:2
      at /test.ts:5:30
````

**Comment by paperclover** [comment]
this is fixed in canary (0.6), you can use `bun upgrade --canary` to update to it.

**Comment by benmerckx** [comment]
Thanks, works indeed
